movierecommender.sql

```diff
@@ -0,0 +1,151 @@
+-- Create Database [movierecommender]
+CREATE database IF NOT EXISTS movierecommender
+DEFAULT CHARACTER SET utf8
+DEFAULT COLLATE utf8mb4_0900_ai_ci;
+
+SET default_storage_engine = InnoDB;
+
+
+-- Specify Database [movierecommender]
+USE movierecommender;
+
+
+-- Create Table [movies_metadata]
+CREATE TABLE `movierecommender`.`movies_metadata` (
+  `adult` BOOLEAN NOT NULL,
+  `belongs_to_collection` varchar(510),
+  `budget` int UNSIGNED NOT NULL,
+  `genres` varchar(510) NOT NULL,
+  `homepage` varchar(255),
+  `id` int NOT NULL,
+  `imdb_id` varchar(255),
+  `original_language` varchar(10),
+  `original_title` varchar(255) NOT NULL,
+  `overview` text,
+  `popularity` DECIMAL(20, 10),
+  `poster_path` varchar(255),
+  `production_companies` text,
+  `production_countries` text,
+  `release_date` DATE,
+  `revenue` bigint,
+  `runtime` DECIMAL(20, 10),
+  `spoken_languages` text,
+  `status` varchar(25),
+  `tagline` text,
+  `title` varchar(255),
+  `video` BOOLEAN,
+  `vote_average` DECIMAL(20, 10),
+  `vote_count` int,
+  PRIMARY KEY (`id`)
+); 
+
+--  Import Data from [movies_metadata.csv] to [movies_metadata]
+load data local infile 'movies_metadata_new.csv' 
```

Comment:
change it to "load data infile  'movies_metadata_new.csv' " since windows machines do not like "local"

Reply:
Deleted "local" in LOAD statement for windows (Kept the one for Mac as comment)